### PR TITLE
Make felinids meow it instead of sayin it.

### PR DESCRIPTION
**Pull request by Adrian16199**
fixes issue #525
A simple PR.

## Why / Balance

Because its nice.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
image

**Changelog**

Make sure to take this Changelog template out of the comment block in order for it to show up.
:cl:
- tweak: Felinids now meow out their words. They can also sigh now.

### CODE REVIEW

File: Resources/Prototypes/Nyanotrasen/Voice/speech_emote_sounds.yml [hunk 1]
```diff
@@ -26,6 +26,8 @@
       collection: FelinidGrowls
     Purr:
       collection: FelinidPurrs
+    Sigh: # Quality of life - Delta V
+      collection: MaleSigh # Quality of life till here - Delta V
```
Comment (DebugOk) [inline Resources/Prototypes/Nyanotrasen/Voice/speech_emote_sounds.yml]: Comments aren't needed in Nyano or Delta files

**Comment by Adrian16199** [comment]
Oh, well, I simply though that maybe if it they would come back then well... i guess they are optional now. Will change it rq.

**Comment by Adrian16199** [comment]
Convertin to draft to add one more thing. (Nevermind)

**Review by Colin-Tel — APPROVED** [review]
🐱

**Pull request merged**